Gemfile

```diff
@@ -27,12 +26,15 @@ gem 'webpacker', '~> 5.0'
 
 gem 'turbolinks', '~> 5'
 
+# newly added
+gem 'credit_card_validations'
+
 
 
 gem 'bootsnap', '>= 1.4.4', require: false
 
 group :development, :test do
-  gem 'rspec-rails', '4.0.0.beta3'
+  gem 'rspec-rails', '~> 4.0'
```

Comment:
This upgraded `rspec-rails` which removed a `name` error for every test when running `rspec`. [[Reference](https://stackoverflow.com/questions/65637389/ruby-on-rails-failure-error-on-all-examples)]

Reply:
do we need to update the rails template for this too?